src/napari/utils/events/event_utils.py

```diff
@@ -17,7 +19,7 @@ def disconnect(self, callback: Callable): ...
 _logger = logging.getLogger(__name__)
 
 
-def disconnect_events(emitter, listener):
+def disconnect_events(emitter: EmitterGroup, listener: object) -> None:
```

Comment:
`listener` is a `Callable`, right? I imagine something like `Callable[[...], None]`?